extension/fts/include/fts-extension.hpp

```diff
@@ -0,0 +1,22 @@
+//===----------------------------------------------------------------------===//
+//                         DuckDB
+//
+// fts-extension.hpp
+//
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "duckdb.hpp"
+
+namespace duckdb {
+
+typedef unsigned char sb_symbol;
```

Comment:
Does this typedef need to be in the header here?

Reply:
It is actually not needed at all anymore now. I think because we managed to compile everything as C++. Removed.